pkg/storaged/client.js

```diff
@@ -258,50 +257,47 @@ export async function btrfs_poll() {
                 // ID 256 gen 7 parent 5 top level 5 path <FS_TREE>/one
                 // ID 257 gen 7 parent 256 top level 256 path one/two
                 // ID 258 gen 7 parent 257 top level 257 path <FS_TREE>/one/two/three/four
-                output = await cockpit.spawn(["btrfs", "subvolume", "list", "-ap", mount_point], { superuser: true, err: "message" });
+                const output = await cockpit.spawn(["btrfs", "subvolume", "list", "-ap", mount_point], { superuser: true, err: "message" });
+                const subvols = [{ pathname: "/", id: 5 }];
+                for (const line of output.split("\n")) {
+                    const m = line.match(/ID (\d+).*parent (\d+).*path (<FS_TREE>\/)?(.*)/);
+                    if (m)
+                        subvols.push({ pathname: m[4], id: Number(m[1]) });
+                }
+                uuids_subvols[uuid] = subvols;
             } catch (err) {
-                console.error(`unable to obtain subvolumes for mount point ${mount_point}`, err);
-                return;
-            }
-            const subvols = [{ pathname: "/", id: 5 }];
-            for (const line of output.split("\n")) {
-                const m = line.match(/ID (\d+).*parent (\d+).*path (<FS_TREE>\/)?(.*)/);
-                if (m)
-                    subvols.push({ pathname: m[4], id: Number(m[1]) });
+                console.warn(`unable to obtain subvolumes for mount point ${mount_point}`, err);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19853-20240118-095029-5e050238-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)